core/src/lib.rs

```diff
@@ -40,3 +40,20 @@ pub use self::from_variant::FromVariant;
 pub use quote::ToTokens;
 #[doc(hidden)]
 pub use syn;
+
+#[macro_export]
+macro_rules! parse_attr_into {
```

Comment:
Would `parse_meta` be a better name for this? While this macro would be helpful for attr macros, it expects a `TokenStream` containing a `NestedMeta` as input, rather than a full attribute.